test/api/v3/unit/models/group.test.js

```diff
@@ -460,73 +457,67 @@ describe('Group Model', () => {
         };
       });
 
-      it('throws an error if no uuids or emails are passed in', (done) => {
-        try {
-          Group.validateInvitations(null, null, res);
-        } catch (err) {
-          expect(err).to.be.an.instanceof(BadRequest);
-          expect(res.t).to.be.calledOnce;
-          expect(res.t).to.be.calledWith('canOnlyInviteEmailUuid');
-          done();
-        }
+      it('throws an error if no uuids or emails are passed in', async () => {
+        await expect(Group.validateInvitations(null, null, res)).to.eventually.be.rejected.and.eql({
+          httpCode: 400,
+          message: 'Bad request.',
```

Comment:
Ok, the `Bad request.` string seems to come from https://github.com/HabitRPG/habitica/blob/develop/website/common/script/libs/errors.js#L31 and it's used as a default when the message isn't provided and it happens because the `res` used in tests (https://github.com/HabitRPG/habitica/pull/8589/files/414e316e832f398b9a695aeb20f9223aa2f556f8#diff-548695dbee775dda2626c88e15cbcec6L459) doesn't return anything but it's just a stub. So false alarm :)